README.md

```diff
@@ -1,55 +1,71 @@
 # Guide
 
-A website to teach Wagtail CMS to content editors, moderators and administrators.
+Guide is a website to help content editors, moderators, administrators, and other users learn how to user the Wagtail content management system (CMS).
 
-Wagtail guide brings you:
-- Tutorials, 
-- How-to, 
-- Reference 
-- Background information
+The Wagtail guide will ultimately include:
+- Tutorials 
+- How-to articles
+- Reference materials
+- Walkthroughs and visual learning materials
 
+# Document contents
 
-## Google Summer of Code
-
-This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/). Find more details about the project [here](https://summerofcode.withgoogle.com/programs/2022/projects/7nMw2hTq).
+- [Installation](#installation)
+- [Contributing](#contributing)
+- [Other Notes](#other-notes)
 
+# Installation
 
 ## Development
 
 We assume that you have basic knowledge of Node/Yarn/Webpack and Python/Django/Wagtail.
 
-### Frontend
+- [Virtualenv](#setup-with-virtualenv)
+- [Gitpod](#setup-with-gitpod)
+
+
+## Setup with Virtualenv
+
+You can run Wagtail Guide locally on your machine using Virtualenv.
+
+#### Dependencies
+- Python 3.9
+- Git
+- [Virtualenv](https://virtualenv.pypa.io/en/latest/installation.html)
+- [VirtualenvWrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html) (optional)
```

Comment:
Since Python 3.3 virtual envs are part of the standard library. That means `virtualenv.pypa.io` and `virtualenvwrapper` are deprecated.

See https://docs.python.org/3/library/venv.html and https://peps.python.org/pep-0405/

On top of that, development environments are preferably as close as possible to a PRD env. And we use `python -m venv ...`